### ISSUE TITLE
Range query error when translating string row keys

### ISSUE THREAD

**Issue by dmibor** [pilosa/pilosa#1678]
For bugs, please provide the following:

### Expected behavior
Range query should be able to translate string row keys for indexes and fields with keys option enabled

### Actual behavior
Getting error
```
{"error":"executing: map reduce: executeRangeShard - reading row: could not convert One of type string to uint64 in Call.UintArg"}
```
when executing Range query on index and field with keys option enabled

### Steps to reproduce the behavior
```
$ curl -XPOST localhost:10101/index/user -d '{"options":{"keys":true}}'
{"success":true}
$ curl localhost:10101/index/user/field/quantity      -X POST      -d '{"options": {"type": "time", "timeQuantum": "YMD", "keys":true}}'
{"success":true}
$ curl localhost:10101/index/user/query -X POST -d 'Set("Column1", quantity="One",2018-07-01T01:00)'
{"results":[true]}
$ curl localhost:10101/index/user/query -X POST -d 'Range(quantity="One",2018-07-01T01:00,2018-07-02T00:00)'
{"error":"executing: map reduce: executeRangeShard - reading row: could not convert One of type string to uint64 in Call.UintArg"}
```

**Comment by travisturner** [comment]
Thanks for the bug report @dmibor!
I pushed a fix and test in PR #1679.

**Comment by travisturner** [comment]
@dmibor this fix is on the `master` branch now. Thanks again for the report, and let us know if you find any other problems.
